tests/CMakeLists.txt

```diff
@@ -51,3 +52,8 @@ if(ENABLE_PERF_TESTS)
 endif()
 
 set(format_sources ${format_sources} PARENT_SCOPE)
+
+get_target_property(TESTS_BIN_UTESTS utest_context RUNTIME_OUTPUT_DIRECTORY)
+get_target_property(TESTS_BIN_PLUGINS plugin_simple RUNTIME_OUTPUT_DIRECTORY)
+file(GENERATE OUTPUT "$<TARGET_FILE_DIR:utest_context>/tests_config2.h"
+    CONTENT "#define TESTS_BIN_UTESTS \"$<TARGET_FILE_DIR:utest_context>\"\n#define TESTS_BIN_PLUGINS \"$<TARGET_FILE_DIR:plugin_simple>\"\n")
```

Comment:
`MSVC: tests: fix paths to utests and test plugins` This is another huge layer of complexity, why not just skip these tests instead?

Reply:
OK. I've added a trivial test for Windows to make sure that everything still works (this helped me before to catch that `:` path separator).